bundles/org.smarthomej.binding.deconz/src/main/java/org/smarthomej/binding/deconz/internal/handler/SensorThermostatThingHandler.java

```diff
@@ -204,10 +203,9 @@ protected void processStateResponse(DeconzBaseMessage stateResponse) {
 
         SensorMessage sensorMessage = (SensorMessage) stateResponse;
         SensorState sensorState = sensorMessage.state;
-        if (sensorState != null && sensorState.windowopen != null && thing.getChannel(CHANNEL_WINDOWOPEN) == null) {
```

Comment:
Why did you remove the check if the channel is already present?

Reply:
It is done twice. The `createChannel` method checks for existent Channel too:

https://github.com/smarthomej/addons/blob/e79f1bd994c01959e9cd902c045c4874123df18a/bundles/org.smarthomej.binding.deconz/src/main/java/org/smarthomej/binding/deconz/internal/handler/DeconzBaseThingHandler.java#L193-L196